### PR TITLE
fixed checklist question

### PR DESCRIPTION
**Pull request by Aagbator**
### Description
Worked on sliderquestionediting.js file

Added aria-live="polite" to the feedback section 

### Screenshots (if appropriate):

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/javascript/ckeditor/checklistquestionediting.js [hunk 1]
```diff
@@ -220,6 +228,57 @@ export default class ChecklistQuestionEditing extends Plugin {
...
+         // the text area label is visible to only screen readers (accessibility purposes)
+         conversion.for( 'upcast' ).elementToElement( {
+            view: {
+                name: 'div',
+                classes: ['text-area-label', 'sr-only']
+            },
+            model: ( viewElement, modelWriter ) => {
+                const id = viewElement.getAttribute('id') || this._nextRetainedDataId();
+                return modelWriter.createElement( 'textareaLabel', {
+                    // HACK: Get the id of the textarea this label corresponds to.
```
Comment (rshipp) [inline app/javascript/ckeditor/checklistquestionediting.js:242]: can you remove all the comments here that are no longer correct?

File: app/javascript/ckeditor/checklistquestionediting.js [hunk 2]
```diff
@@ -220,6 +228,57 @@ export default class ChecklistQuestionEditing extends Plugin {
...
+                } );
+            }
+        } );
+        conversion.for( 'editingDowncast' ).elementToElement( {
+            model: 'textareaLabel',
+            view: ( modelElement, viewWriter ) => {
+                const id = modelElement.getAttribute('id') || this._nextRetainedDataId();
+                const label = viewWriter.createEditableElement( 'div', {
+                    // NOTE: We don't set the 'for' attribute in the editing view, so that clicking in the label
+                    // editable to type doesn't also toggle the checkbox.
```
Comment (rshipp) [inline app/javascript/ckeditor/checklistquestionediting.js:265]: more outdated comments

**Comment by rshipp** [comment]
good to go after removing the old comments

File: app/javascript/ckeditor/checklistquestionediting.js (hunk 1 above)
Comment (Aagbator) [reply app/javascript/ckeditor/checklistquestionediting.js]: fixed

**Review by rshipp — APPROVED** [review]

**Pull request merged**